packages/strapi-global-object/strapi-global.ts

```diff
@@ -0,0 +1,20 @@
+import knex from "knex";
+
+const strapiObject = {
+  plugins: {},
+  components: {},
+  contentTypes: {},
+  query: (s: any) => s,
```

Comment:
why any? please make proper object, I think there is something like model or contentType `name` and it doesn't return argument itself, 
AFAIK if You `strapi.query('article') then it will return a Article entity with model inside it

Reply:
fixed